volatility3/framework/plugins/windows/svcdiff.py

```diff
@@ -0,0 +1,83 @@
+# This file is Copyright 2024 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+# This module attempts to locate skeleton-key like function hooks.
+# It does this by locating the CSystems array through a variety of methods,
+# and then validating the entry for RC4 HMAC (0x17 / 23)
+#
+# For a thorough walkthrough on how the R&D was performed to develop this plugin,
+# please see our blogpost here:
+#
+# https://volatility-labs.blogspot.com/2021/10/memory-forensics-r-illustrated.html
+
+import logging
+
+from volatility3.framework import symbols
+from volatility3.framework.configuration import requirements
+from volatility3.plugins.windows import svclist, svcscan
+from volatility3.framework.symbols.windows import versions
+
+vollog = logging.getLogger(__name__)
+
+
+class SvcDiff(svclist.SvcList, svcscan.SvcScan):
```

Comment:
I was looking for the run method, and then realize this doesn't inherit directly from `PluginInterface`, but uses multiple inheritance!  5:S  This makes individual plugins much more fragile to changes in the underlying code and requires updating multiple plugins depending on the changes.  Unless there's a very good reason, I'd strongly suggest exposing the functions that are needed in each inheritted plugin, and call them explicitly.  It'll also help other plugins reuse them properly and keep the dependency tree functioning properly.

Reply:
Fixed to match modules/modscan